tenacity/_utils.py

```diff
@@ -133,27 +93,4 @@ def get_callback_name(cb):
         return ".".join(segments)
 
 
-try:
-    now = time.monotonic  # noqa
-except AttributeError:
-    from monotonic import monotonic as now  # noqa
-
-
-class cached_property(object):
```

Comment:
not used at code base -> no reason to migrate on backports or make semi-stdlib code